src/views/bootsources/list/hooks/useBootableVolumesColumns.ts

```diff
@@ -0,0 +1,65 @@
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { K8sResourceCommon, TableColumn } from '@openshift-console/dynamic-plugin-sdk';
+import { sortable } from '@patternfly/react-table';
+
+import { getDataSourcePreferenceLabelValue } from '../../utils/utils';
+
+const useBootableVolumesColumns = (): TableColumn<K8sResourceCommon>[] => {
+  const { t } = useKubevirtTranslation();
+
+  return [
+    {
+      title: t('Name'),
+      id: 'name',
+      transforms: [sortable],
+      sort: 'metadata.name',
+    },
+    {
+      title: t('Operating system'),
+      id: 'os',
+    },
+    {
+      title: t('Description'),
+      id: 'description',
+      transforms: [sortable],
+      sort: 'metadata.annotations.description',
+    },
+    {
+      title: t('Preference'),
+      id: 'preference',
+      transforms: [sortable],
+      sort: (dataSources, dir) => {
+        return dataSources.sort((a, b) => {
+          const aUpdated = getDataSourcePreferenceLabelValue(a);
+          const bUpdated = getDataSourcePreferenceLabelValue(b);
+
+          if (aUpdated && bUpdated) {
+            return dir === 'asc'
+              ? aUpdated.localeCompare(bUpdated)
+              : bUpdated.localeCompare(aUpdated);
+          }
+        });
+      },
+    },
+    {
+      title: t('Namespace'),
+      id: 'namespace',
+      transforms: [sortable],
+      sort: 'metadata.namespace',
+    },
+    {
+      title: t('Created at'),
+      id: 'created',
+      transforms: [sortable],
+      sort: 'metadata.creationTimestamp',
+    },
+
+    {
+      title: t(''),
```

Comment:
the translate here is intentional?

Reply:
This should have been representing the Actions column. And that's WIP right now in here. Sorry for not commenting this properly.